torchbenchmark/models/LearningToPaint/__init__.py

```diff
@@ -1,99 +1,99 @@
+            'validate_interval': 1,  # Allows us to capture the discriminator model in the 1st iteration.
+            'max_step': 1,
+            'env_batch': 64,
+            'batch_size': 96,
+            'discount': 0.95**5,
+            'episode_train_times': 10,
+            'noise_factor': 0.0,
+        })
+        # Input image from CelebFaces are resized to 128 x 128.
+        # Create 2000 random tensors for input, but fastenv will still load 200,000 images.
+        self.width = 128
+        self.image_examples = torch.rand(2000, 3, self.width, self.width)
+
+        # LearningToPaint includes actor, critic, and discriminator models, make sure to run all of them!
+        self.env = fastenv(max_episode_length=self.args.max_step, env_batch=self.args.env_batch,
+                           images=self.image_examples, device=self.device)
+        self.agent = DDPG(batch_size=self.args.batch_size, env_batch=self.args.env_batch, max_step=self.args.max_step,
+                          discount=self.args.discount, device=self.device)
+        self.evaluate = Evaluator(args=self.args, writer=None)
         self.step = 0
-        self.opt = Namespace(**{
-            'batch_size': 64,
-            'debug': '',
-            'script': False,
-            })
-
-        train_batch = []
-        self.ground_truth = []
-        for i in range(self.opt.batch_size):
-            f = np.random.uniform(0, 1, 10)
-            train_batch.append(f)
-            self.ground_truth.append(draw(f))
-
-        train_batch = torch.tensor(train_batch).float()
-        self.ground_truth = torch.tensor(self.ground_truth).float()
-
-        net = net.to(self.device)
-        self.eval_model = net.to(self.device)
-        # model needs to in `eval`
-        # in order to be optimized for inference
-        self.eval_model.eval()
-        train_batch = train_batch.to(self.device)
-        self.ground_truth = self.ground_truth.to(self.device)
-
-        if self.jit:
-            if hasattr(torch.jit, '_script_pdt'):
-                net = torch.jit._script_pdt(net, example_inputs = [(train_batch, ), ])
-                self.eval_model = torch.jit._script_pdt(self.eval_model)
-            else:
-                net = torch.jit.script(net, example_inputs = [(train_batch, ), ])
-                self.eval_model = torch.jit.script(self.eval_model)
-            self.eval_model = torch.jit.optimize_for_inference(self.eval_model)
-
-
-        self.module = net
-        self.example_inputs = train_batch
-        self.optimizer = optim.Adam(self.module.parameters(), lr=3e-6)
+        self.observation = self.env.reset()
+        self.agent.reset(self.observation, self.args.noise_factor)
 
     def get_module(self):
-        return self.module,[self.example_inputs]
-
-    # LearningToPaint has another model
-    # instance for inference that has
-    # already been optimized for inference
-    def set_eval(self):
-        pass
+        raise NotImplementedError()
```

Comment:
This API has some users (e.g., https://github.com/pytorch/benchmark/pull/479), can you implement this (e.g. using randomly generate inputs to the actor model)?

Reply:
Sure, I can take a further look into this.

Reply:
done!